.github/workflows/publish-master.yaml

```diff
@@ -71,9 +71,18 @@ jobs:
         run: |
           docker login -u "${{ secrets.QUAYIO_USERNAME }}" -p "${{ secrets.QUAYIO_PASSWORD }}" quay.io
           set -x
+
           docker tag monocle_web:latest quay.io/change-metrics/monocle_web:latest
           docker push quay.io/change-metrics/monocle_web:latest
-          docker tag monocle_api:latest quay.io/change-metrics/monocle_backend:latest
-          docker push quay.io/change-metrics/monocle_backend:latest
-          docker tag monocle_api-ng:latest quay.io/change-metrics/monocle_api:latest
+
+          docker tag monocle_api:latest quay.io/change-metrics/monocle_api:latest
           docker push quay.io/change-metrics/monocle_api:latest
+
+          docker tag monocle_api-legacy:latest quay.io/change-metrics/monocle_api-legacy:latest
+          docker push quay.io/change-metrics/monocle_api-legacy:latest
+
+          docker tag monocle_crawler:latest quay.io/change-metrics/monocle_crawler:latest
+          docker push quay.io/change-metrics/monocle_crawler:latest
+
+          docker tag monocle_crawler-legacy:latest quay.io/change-metrics/monocle_crawler-legacy:latest
+          docker push quay.io/change-metrics/monocle_crawler-legacy:latest
```

Comment:
We currently have monocle_api for the new api and macroscope, and monocle_backend for the legacy api and crawler. If you want to change the image, then we need to update the compose, but I think we can keep the current names.